src/profile/alyssa-gerhart.js

```diff
@@ -0,0 +1,35 @@
+function getStudents() {
+    const studentsApiUrl = "http://localhost:4000/students?slug=alyssa-gerhart";
+    return fetch(studentsApiUrl, {
+      method: "GET"
+    })
+      .then((response) => response.json())
+      .then((data) => data)
+      .catch((error) => console.error(error));
+  }
+  
+document.addEventListener("DOMContentLoaded", async () => {
+ const students = await getStudents();
+  
+    const studentContainer = document.getElementById("student-cards-container");
+  
+    let content = "";
+    students.forEach((student) => {
+      content += `
+      <div class="card linked">
+        <div class="card-header">
+          <img class="card-avatar" src="alyssa-avatar.JPG" />
```

Comment:
I see why you did this. At the original path, it did't just work... BUT we can do this

```suggestion
          <img class="card-avatar" src="../${student.avatar}" />
```
The idea here is that we want to leverage the API as much as possible to render out interface. If this was a real API, `avatar` would just a link to the image assuming you had uploaded somewhere. The avatar images wouldn't even be in the repository.

What we should have done since the beginning was set the `avatar` field to just the name of the file like:
 ```
 avatar": "my-avatar.png"
```
instead of 

```
avatar": "public/student-avatars/my-avatar.png"
```

So that we just provide the path wherever we use it. That's not a change for you though, just thinking out loud.